olive/engine/engine.py

```diff
@@ -249,7 +236,7 @@ def run(
                 result_name = f"{prefix_output_name}metrics"
                 results_path = output_dir / f"{result_name}.json"
                 with open(results_path, "w") as f:
-                    json.dump(results, f, indent=4)
+                    f.write(results.json())
```

Comment:
I like this since it would remove the "__root__", right?

Reply:
Yes~